ui/src/app/richskill/import/batch-import.component.ts

```diff
@@ -539,5 +553,76 @@ export class BatchImportComponent extends QuickLinksHelper implements OnInit {
   handleSimilarityOk(importSimilar: boolean): void {
     this.importSimilarSkills = importSimilar
   }
+
+  protected handleClickAddToWorkspace(): void {
+    let skillListUpdate = new ApiSkillListUpdate({
+      add: new ApiSearch(
+        {uuids:this.skillsToBeImported?.map(skill => skill.uuid)}
+      )
+    })
+    this.toastService.showBlockingLoader()
+    this.collectionService.getWorkspace().subscribe(workspace => {
+      this.collectionService.updateSkillsWithResult(workspace.uuid, skillListUpdate, undefined).subscribe(result => {
+        if (result) {
+          const message = `You added ${result.modifiedCount} RSDs to your workspace.`
+          this.toastService.showToast("Success!", message)
+          this.toastService.hideBlockingLoader()
+        }
+      })
+    })
+  }
+
+  protected handleAddToExistingCollection() {
+    this.router.navigate(["/collections/add-skills"],
+      {state:{selectedSkills: this.skillsToBeImported, totalCount:this.skillsToBeImported?.length}}
+    )
+  }
+
+  protected handleAddToANewCollection() {
```

Comment:
What is the destination for these skills? What is the name of the new collection? Either the user should be prompted to provide a name, or one should be generated and given to the user.